### DISCUSSION TITLE
Placeholder for maintenance

### DISCUSSION THREAD

**Discussion by JaTiTV** [Zrips/CMI#5414] [Enhancement]
Could you add a placeholder that you can display for example in a hologram or in another plugin that supports PlaceholderAPI if a server from the network is in maintenance mode or not? Or does this already exist? For example %cmi_maintenance_<server>%

**Comment by CloudeLecaw** [comment]
Why a placeholder?
Why not simply design your hologramm to show this text when you maintain a server? You can create a lovly customalais for that, that checks what Servername u used and react in the correct hologramm for each.
This also allow you to copy design the full msg After and for each server into the Hologramm

**Reply by JaTiTV** [reply]
A placeholder would be useful since we use InteractiveBoard among other things. And I think that with a placeholder you have more possibilities instead of changing the texts in several configs manually.
